src/Components/Home/Home.css

```diff
@@ -26,10 +19,39 @@
   text-align: center;
   font-size: 1rem;
   margin: 1rem;
-  border: 1px solid black;
+  border: 1px solid var(--black);
   border-radius: 5px;
+  color: var(--black);
+}
+
+.token input::placeholder {
+  color: var(--darkBlue);
+  opacity: 0.8;
 }
 
-.token button.joinExistingList {
-  width: 210px;
+@media only screen and (min-width: 900px) {
+  .HomeButton {
+    font-size: 1.2rem;
+    width: 20rem;
+  }
+
+  .token button.joinExistingList {
+    width: 20rem;
+  }
```

Comment:
Both of the buttons on the Home page have the `HomeButton` class so the `.token button.joinExistingList` declaration is not doing anything.

Also don't know if it's intentional but the new token button and join existing list button have different widths once the screen is below 900px.